Cargo.toml

```diff
@@ -96,6 +96,10 @@ cargo = { path = "src/tools/cargo" }
 rustfmt-nightly = { path = "src/tools/rustfmt" }
 
 [patch.crates-io]
+fortanix-sgx-abi = { git = "https://github.com/fortanix/rust-sgx", branch = "raoul/edmm" }
+sgx-isa = { git = "https://github.com/fortanix/rust-sgx", branch = "raoul/edmm" }
```

Comment:
Patches need to be removed when https://github.com/fortanix/rust-sgx/pull/292 merges